src/plugins/data/server/search/search_service.ts

```diff
@@ -75,6 +75,12 @@ export class SearchService implements Plugin<ISearchSetup, void> {
     const api: ISearchSetup = {
       registerSearchStrategyContext: this.contextContainer!.registerContext,
       registerSearchStrategyProvider,
+      createScopedSearchApi: caller => {
+        return createApi({
+          caller,
+          searchStrategies: this.searchStrategies,
+        });
+      },
```

Comment:
This allows us to expose a `search` api that can be used without a RouteContext, as alerting runs in Tasks rather than via Routed requests.